commands.go

```diff
@@ -72,6 +73,7 @@ func parseCommand() (Command, error) {
 	return v, nil
 }
 
+// ListCommand contains the information required to contruct a call to list certificates
```

Comment:
Typo (contruct vs construct)

Reply:
It's almost like I copied and pasted that...